meson_options.txt

```diff
@@ -1,4 +1,6 @@
 option('bootdir', type : 'string', value : '/boot/efi', description : 'Directory for EFI system partition')
+option('polkit', type : 'boolean', value : true, description : 'enable PolicyKit support')
```

Comment:
Since this is affecting the installation of the daemon and not just the installation of the rules I think this string should be renamed to "standalone".  It can impact anything it needs to then that doesn't already have it's own string.

Reply:
Usually configure options are default on, and let you disable some action. I think a -Ddaemon=false and then checking for an option makes more sense than a -Dstandalone and then checking for the negative of the option. I've fixed up the commit to add a daemon=false mode -- does that look sane?

Reply:
I think that logic makes sense, yeah.